package.json

```diff
@@ -1,20 +1,58 @@
 {
   "name": "css-frameworks-ca",
-  "version": "1.0.0",
+  "version": "1.1.0",
   "description": "Test",
+  "type": "module",
   "main": "index.js",
   "scripts": {
     "build": "sass src/scss:dist/css",
-    "start": "sass --watch src/scss:dist/css & live-server"
+    "watch": "sass --watch src/scss:dist/css",
+    "start": "live-server",
+    "dev": "npm-run-all --parallel watch start",
+    "format": "prettier -w src/**/*.js",
+    "lint": "eslint src/**/*.js",
+    "lint-fix": "eslint src/**/*.js --cache --fix",
+    "prepare": "husky install",
+    "test": "npm run test-unit && npm run test-e2e-cli",
+    "test-unit": "jest",
+    "test-e2e": "cypress open",
+    "test-e2e-cli": "cypress run"
   },
   "keywords": [],
   "author": "Noroff",
   "license": "ISC",
   "devDependencies": {
-    "live-server": "^1.2.2",
+    "@babel/core": "^7.19.3",
+    "@babel/preset-env": "^7.19.4",
+    "@eslint/js": "^9.9.0",
+    "cypress": "^13.13.2",
+    "eslint": "^9.9.0",
+    "eslint-plugin-cypress": "^3.5.0",
+    "eslint-plugin-jest": "^28.8.0",
+    "globals": "^15.9.0",
+    "husky": "^9.1.4",
+    "jest": "^29.2.0",
+    "lint-staged": "^15.2.7",
+    "live-server": "^1.2.0",
+    "npm-run-all": "^4.1.5",
+    "prettier": "^3.3.2",
     "sass": "^1.54.8"
   },
   "dependencies": {
-    "bootstrap-dark-5": "^1.1.3"
+    "@typescript-eslint/utils": "^8.1.0",
```

Comment:
Why do you need this in a js project? Consider removing unrelated files and configuration